jira/client.py

```diff
@@ -2807,9 +2807,17 @@ def delete_user_avatar(self, username, avatar):
         return self._session.delete(url, params=params)
 
     def search_users(
-        self, user, startAt=0, maxResults=50, includeActive=True, includeInactive=False
+        self,
+        user=None,
+        startAt=0,
+        maxResults=50,
+        includeActive=True,
+        includeInactive=False,
+        query=None,
     ):
         """Get a list of user Resources that match the specified search string.
+        "username" query parameter is deprecated; the expected parameter now is "query", which can just be the full
```

Comment:
Refer to my previous comment. We should specify here that this only applied to "Cloud" instances, at least for the time being.

```suggestion
        "username" query parameter is deprecated in Jira Cloud; the expected parameter now is "query", which can just be the full
```